src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public void getStatistic (String fromFileName, String toFileName) throws FileNotFoundException {
+        File file = new File(fromFileName);
+        String stringSupply = "supply";
+        String stringBuy = "buy";
+        String[] arrayString;
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        String myResult;
+
+        try {
+            List<String> stringList = Files.readAllLines(file.toPath());
+            for (String string : stringList) {
+                arrayString = string.split(",");
+                    if (arrayString[0].equals(stringSupply)) {
+                        supply += Integer.valueOf(arrayString[1]);
```

Comment:
Using `Integer.valueOf` is not necessary here. Consider using `Integer.parseInt(arrayString[1])` for parsing the string to an integer, which is more common for primitive int operations.